src/components/chip-set/chip-set.tsx

```diff
@@ -208,16 +196,19 @@ export class ChipSet {
     @State()
     private inputChipIndexSelected: number = null;
 
-    private mdcChipSet: MDCChipSet;
+    @State()
+    private selectedChip: Chip;
+
+    @State()
+    private selectedChipIds: Array<string | number>;
```

Comment:
Also, what does "selected" mean? Does it mean that the chip is highlighted, so that pressing backspace removes it, or does it mean that the chip is "in" the chip-set? Or something else?

Reply:
For `choice` and `filter`, I guess it's pretty straightforward what "selected" means. I had forgotten that we had those variants 😅

Does "selected" have any meanings other than that as well?

Reply:
`selected` mean when `chip.select = true`

Reply:
selected also adds styles on `limel-chip`

Reply:
He is asking why we add a prefix `selected` in `selectedChipIds` 😅

Reply:
> He is asking why we added a prefix `selected` in `selectedChipIds` 😅

Not so much why you added the prefix, as: _What does it signify that a chip is "selected"?_

As I mentioned before, I had forgotten that a chip-set could be anything other than the `input` type 🙈

My question was more, does it mean all of these things?

<img width="534" alt="A 'choice' chip-set, with the chips 'Lime', 'Apple', and 'Banana'. The 'Apple' chip looks like it's pressed down and is rendered with green text" src="https://github.com/Lundalogik/lime-elements/assets/154725/3ae42c73-8676-464a-9399-b7ed21a7ccea">

<img width="534" alt="A 'filter' chip-set, with the chips 'Lime', 'Apple', and 'Banana'. The 'Apple' chip looks like it's pressed down and is rendered with green text, and a green 'light' in the bottom left corner" src="https://github.com/Lundalogik/lime-elements/assets/154725/0c4ca858-4751-41a3-a062-62ebd0c00eca">

<img width="534" alt="An 'input' chip-set, with the chips 'Elephant', 'Caterpillar', 'Badger', and 'Fish'. The 'Caterpillar' chip has a red outline" src="https://github.com/Lundalogik/lime-elements/assets/154725/ece0d4ef-3efe-4781-8a40-7ba7fedcc23a">

The first two I could answer "yes" by looking at the examples in the docs, and the last one I found out is also "yes", by looking a little closer at the code.

I felt like at least the last example, with the red outline, is quite different from the others. If it was a completely new addition, I would have probably complained about it, but since it's just an adaptation of `chip.selected`, and that already worked like this, it's ok.

Reply:
I consider this question resolved, so I'll mark the thread as such 🙂